### ISSUE TITLE
SDSC: PKG - expanse/0.17.3/gpu/b - Missing  gromacs GPU(example application)

### ISSUE THREAD

**Issue by nwolter** [sdsc/spack#54]
gromacs/2019.6
gromacs/2019.6-plumed

**Comment by mkandes** [comment]
Two variants are available in `expanse/017.3/gpu/a`.

```
[mkandes@login02 ~]$ module use /cm/shared/apps/spack/0.17.3/gpu/a/share/spack/lmod/linux-rocky8-x86_64/Core
[mkandes@login02 ~]$ module spider gromacs/2020.4

----------------------------------------------------------------------------
  gromacs/2020.4:
----------------------------------------------------------------------------
     Versions:
        gromacs/2020.4/eglkgfz
        gromacs/2020.4/hyq22f5

----------------------------------------------------------------------------
  For detailed information about a specific "gromacs/2020.4" package (including how to load the modules) use the module's full name. Note that names that have a trailing (E) are extensions provided by other modules.
  For example:

     $ module spider gromacs/2020.4/hyq22f5
----------------------------------------------------------------------------

[mkandes@login02 ~]$
```

**Comment by nwolter** [comment]
Test:  OK
sample script located at:
/expanse/lustre/projects/use300/nickel/2023/gromacs/gpu/gromacs-1node-4GPU_gcc.sb
